libandroid-navigation/src/test/java/com/mapbox/services/android/navigation/v5/offroute/OffRouteDetectorTest.java

```diff
@@ -43,7 +43,6 @@ public void setup() throws Exception {
 
     offRouteDetector = new OffRouteDetector();
     offRouteDetector.setOffRouteCallback(mockCallback);
-    setupStepPoints(offRouteDetector);
   }
 
```

Comment:
Overall same comments pointed out above ☝️

Reply:
Not sure what the comment above here means